lib/ts/recipe/session/with-jwt/recipeImplementation.ts

```diff
@@ -174,16 +186,14 @@ export default function (
 
             let existingJWTValidity = decodedPayload.exp - currentTimeInSeconds;
 
-            let defaultClaimsToAdd = {
+            newAccessTokenPayload = {
                 sub: sessionInformation.userId,
                 iss: appInfo.apiDomain.getAsStringDangerous(),
+                ...newAccessTokenPayload,
```

Comment:
The `newAccessTokenPayload` may also have `_jwtPName` in it.. so we need to make sure that those are removed.

Reply:
Logic has been refactored to always remove the old jwt and property name before creating a new JWT